src/Jackett.Common/Indexers/Hebits.cs

```diff
@@ -100,60 +102,58 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             var response = await RequestStringWithCookies(searchUrl);
             try
             {
-                CQ dom = response.Content;
+                var parser = new HtmlParser(); var dom = parser.ParseDocument(response.Content);
 
-                var qRows = dom[".browse > div > div"];
+                var rows = dom.QuerySelectorAll(".browse > div > div");
 
-                foreach (var row in qRows)
+                foreach (var row in rows)
                 {
                     var release = new ReleaseInfo();
 
-                    var qRow = row.Cq();
-
-                    var debug = qRow.Html();
+                    var debug = row.Html();
```

Comment:
Is there a alternative to `.Html()` ?
The same we are using `TextContent` instead of `Text()`

Reply:
Just checked and yes, `.Html()` doc description says it gets InnerHtml. So use `.InnerHtml` directly.